geode-core/src/test/java/org/apache/geode/distributed/internal/ServerLocationTest.java

```diff
@@ -43,5 +43,7 @@ public void objectCreatedWithCopyConstructorwhenComparedToOrigin_isEqual() {
     final ServerLocation serverLocation2 = new ServerLocation(serverLocation1);
 
     assertThat(serverLocation1).isEqualTo(serverLocation2);
+    assertThat(serverLocation1.getRequiresCredentials())
+        .isEqualTo(serverLocation2.getRequiresCredentials());
```

Comment:
I meant that in the constructor if statement determines how the value is set but there is only one test for this constructor. There should be two tests, one that exercised the branch then block and one that does not.